### PR TITLE
Issue-#1193: Simple flow v4

### LINKED ISSUE #1193: "Simple Flow" README works in v4
The v4 version of this flow
https://github.com/oceanprotocol/ocean.js#simple-flow should run end-to-end without errors.

Write the README as integration tests, and if needed, underlying components.
We can use some of these examples: https://github.com/akshay-ap/Ocean.js-PlayGround/tree/v4

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #1193

Changes proposed in this PR:

- Update `quickstart_simple.md`

### CODE REVIEW

File: docs/quickstart_simple.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Here are the the steps:
 1. Initialize services
 2. Create a new node.js project
 3. Install dependencies
-4. Create a config file and update contract addresses
-5. Publish a new datatoken
+4. Create a .env and config file
+5. Publish a new dataNFT and datatoken
```
Comment (kremalicious) [inline docs/quickstart_simple.md:11]:
```suggestion
5. Publish a new Data NFT and Datatoken
```

File: docs/quickstart_simple.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ Open the package.json file in a text editor and update the dependencies to inclu
 
 ```JSON
   "dependencies": {
-    "@oceanprotocol/contracts": "^0.5.6",
-    "@oceanprotocol/lib": "^0.6.5",
-    "web3": "^1.3.0"
+    "@oceanprotocol/contracts": "^0.5.6", // TODO: replace version
```
Comment (kremalicious) [inline docs/quickstart_simple.md:47]: this is so wrong, forcing us to constantly update version number in here. The standard to tell people to install things in js ecosystem is NOT to tell people to edit the `package.json` but using the `npm install` command.

In this case the instructions should be:

```
npm install @oceanprotocol/lib web3
```

And that's it.

File: docs/quickstart_simple.md [hunk 3]
```diff
@@ -58,7 +58,21 @@ npm install
 
 ## 4. Create a config file and update contract addresses
 
-Create a new config.js file:
+### Create a .env file
+
+```bash
+NETWORK_URL=http://172.15.0.3:8545
```
Comment (kremalicious) [inline docs/quickstart_simple.md:64]: what are these URLs?

File: docs/quickstart_simple.md [hunk 4]
```diff
@@ -67,37 +81,49 @@ cat > config.js
...
 const { ConfigHelper } = require("@oceanprotocol/lib");
-const Web3 = require("web3");
-const defaultConfig = new ConfigHelper().getConfig("development");
 
-const urls = {
-  networkUrl: "http://localhost:8545",
-  aquarius: "http://localhost:5000",
-  providerUri: "http://localhost:8030",
-};
+var oceanConfig = new ConfigHelper().getConfig(process.env.OCEAN_NETWORK);
```
Comment (kremalicious) [inline docs/quickstart_simple.md:90]: either use `const/let` or the old `var`. But never mix them all in one file. So this should be a `const`

File: docs/quickstart_simple.md [hunk 5]
```diff
@@ -67,37 +81,49 @@ cat > config.js
 Now open the config.js in your code editor and enter the following:
 
 ```Javascript
+require('dotenv').config()
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+const fs = require("fs");
```
Comment (kremalicious) [inline docs/quickstart_simple.md:86]: consistency: use either single, or double quotes but never mix them all in one file. If you have prettier installed in your editor it should auto format code snippets in markdown files based on our eslint config

File: docs/quickstart_simple.md (hunk 3 above)
Comment (mihaisc) [reply docs/quickstart_simple.md:64]: I believe these are urls from barge . `NETWORK_URL` is the rpc node url

Comment (akshay-ap) [reply docs/quickstart_simple.md:54]: yes. It is the rpc node url

**Comment by AnaLoznianu** [comment]
@miquelcabot @jamiehewitt15 @akshay-ap I think we can close this as we have this PR ready: https://github.com/oceanprotocol/ocean.js/pull/1438 which takes a different approach to the READMEs

**Comment by jamiehewitt15** [comment]
The new plan is to reduce the ocean.js documentation down to one readme and one code examples file (possibly split into two when we add compute to data) as described [here](https://github.com/oceanprotocol/ocean.js/issues/1354#issuecomment-1112032587)

**Pull request closed without merging**